### ISSUE TITLE
HueImmersive and Hue Firmware 1.12 doesn't work?

### ISSUE THREAD

**Issue by DKelnozz** [Blodjer/HueImmersive#18]
Yesterday I had to reset my old bridge (1.0) but since then I can't paire it again with HueImmersive and some other apps on my phone. After some searching, I found this about the new Firmware http://www2.meethue.com/en-us/support/firmware-update/ Is the failure on my site and I have to keep on trying to pair or does the new Firmware locked HueImmersive out.
Thanks for your help and sorry for my bad English, I'm not a native English speaker.

**Comment by MichaelSchemmer** [comment]
Try this out http://www.screenbloom.com/

**Comment by Blodjer** [comment]
Hi, seems like they changed the authentication process.
What the fuck Philips? This affects all apps. I don't have any time to fix it so HueImmersive and many other apps are unusable now. Nevertheless, it still works for the users who used HueImmersive before the update.
Sorry

**Comment by DKelnozz** [comment]
No problem and thanks for the replies.

I managed to buy a new old bridge in the last days, so HueImmersive works for me again, but I will definetly try out screenbloom.

Have nice weekend both of you.

**Comment by polykoma** [comment]
Hi my friends.
Hi Blodjer

Im so sad i updated some weeks ago. I didnt used HueImmersive due a move to a new home.

now im kinda screwed. i really love your tool most and its so sad you dont have time for the adjustments.
please keep your lovely tool alive for us!

kindest regards
poly

**Comment by lachmans** [comment]
Polykoma, I'm in the exact same position that you're in. I've asked Blodjer to update the app, I even told him i'm willing to pay for a fix but to bad. He said he was to busy to update the app.

**Comment by DKelnozz** [comment]
The only hint I can give you, is to look for an old hue bridge (version 1.0) and constantly ignore the demand of the app for an update. I bought one on ebay for 15€ and sold my updated hue bridge in return.

**Comment by polykoma** [comment]
Did anyone tried screenbloom mentioned by another user above ?

I really dont want to change the bridge since i have many lamps and remotes to configure again then :/

**Comment by DKelnozz** [comment]
Yes I did and also tried Albedo (http://loginer.net/albedo/) but for me HueImmersive were the better solution.

**Comment by polykoma** [comment]
Will it work with the latest firmware then ? :)
